docs/books/nvchad/vale_nvchad.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Using vale in NvChad
+author: Steven Spencer
+contributors: Franco Colussi
+tags:
+  - vale
+  - linters
+---
+
+# `vale` in NvChad (Neovim)
+
+## Introduction
+
+`vale.sh` is one of the foremost open source projects for technical writers looking to improve their voice and style consistency. It can be used with a number of editors on nearly every major OS platform (Linux, MacOS, Windows). You can find out more about the project by heading up to [the vale.sh website](https://vale.sh/). This guide is going to walk you through adding `vale` to NvChad. Since it is included in the Mason packages for install, the process is not to difficult, although it does involve some minor editing and configuration to get things going. To be clear, NvChad is really the configuration manager for the editor Neovim, so from this point forward, we will be referencing `nvim`.
+
+## Prerequisites
+
+* Familiarity with NvChad 2.0 is helpful
+* Ability to change files from the command line by using your favorite editor. (`vi` or your favorite)
+
+## Installing `vale` using Mason
+
+Installing `vale` from within NvChad using Mason, will allow for you to stay up-to-date with the package without a whole lot of extra steps. Running Mason periodically from the within `nvim` will show you if there are updates that need to be installed and allow you to update them from there. This includes `vale` once it is installed. Let us start by running `nvim` to pull up an empty file and then get into the command mode by using <kbd>SHIFT</kbd>+<kbd>:</kbd>+Mason, which should show an interface similar to this one:
```

Comment:
streamline word count in first sentence by reframing as active tense and positive: "...will keep the package up-to-date with just a few extra steps."